### ISSUE TITLE
[REQUEST] Seed sampling for probability density distribution

### ISSUE THREAD

**Issue by admk** [elxris/Turnip-Calculator#6]
Is it possible to sample multiple runs to predict the probability density of future turnip prices?

**Comment by mtaylor76** [comment]
I'm not sure there's enough bits of data in each week of price data to determine the seed accurately enough to calculate the PDF for future prices. There was a post on /r/acturnips that explained it better, let me see if I can find it.

**Comment by elxris** [comment]
Yeah, this is something I have in scope, a bit different, but more real.

Each pattern has it's own probability. And we can assign that weight without calculating everything.

**Comment by elxris** [comment]
Please review this comment: https://github.com/elxris/Turnip-Calculator/issues/115#issuecomment-629938533 and let me know what you think.
